core/src/main/resources/lib/form/breadcrumb-config-outline.jelly

```diff
@@ -29,5 +29,5 @@ THE SOFTWARE.
   </st:documentation>
 
   <st:adjunct includes="lib.form.breadcrumb-config-outline.init"/>
-  <l:breadcrumb title="${%configuration}" id="inpage-nav" />
+  <l:breadcrumb title="${%Configure System}" id="inpage-nav" />
```

Comment:
This is a general purpose tag, nothing about it says it's necessarily the global configuration only (and it's used in at least one plugin). While it's not currently used there, it probably should be used in Global Tool Configuration and Global Security Configuration. See https://github.com/jenkinsci/jenkins/pull/6068 for a different approach to get a nicer label.